dist/scripts/scripts.js

```diff
@@ -1423,6 +1425,11 @@ return n(r);
 }, 1e3);
 } ]).run([ "IS_IOS", function(e) {
 e && $("body").addClass("ios");
+} ]).run([ "$rootScope", "APIService", function(e, t) {
```

Comment:
is this code generated?

Reply:
yes, everything in `dist` is generated and not excluded by gitignore